pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -130,23 +131,37 @@ func RunStartBuild(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra
 		return cmdutil.UsageError(cmd, "Must pass a name of a build config or specify build name with '--from-build' flag")
 	}
 
-	name := buildName
-	isBuild := true
-	if len(name) == 0 {
-		name = args[0]
-		isBuild = false
+	namespace, _, err := f.DefaultNamespace()
+	if err != nil {
+		return err
 	}
 
-	if webhooks.Provided() {
-		return RunListBuildWebHooks(f, out, cmd.Out(), name, isBuild, webhooks.String())
+	var (
+		name     = buildName
+		resource = "builds"
+	)
+
+	if len(name) == 0 && len(args[0]) > 0 {
```

Comment:
the logic above that guarantees we have a single arg in this case is buried enough that I'd like an explicit `len(args) > 0` check here